core/domain/classifier_domain.py

```diff
@@ -286,6 +286,17 @@ def update_status(self, status):
 
         self._status = status
 
+    def update_training_data(self, training_data):
```

Comment:
Actually -- perhaps I am confused too. I thought we weren't supposed to be updating the training data after the initial classifier training job model was created?